saleor/settings.py

```diff
@@ -506,6 +506,7 @@ def get_host():
         "saleor.graphql.middleware.OpentracingGrapheneMiddleware",
         "saleor.graphql.middleware.JWTMiddleware",
         "saleor.graphql.middleware.app_middleware",
+        "graphene_django.debug.DjangoDebugMiddleware",
```

Comment:
Is this related to this PR?

Reply:
We should remove it. It's a legacy of optimization.

Reply:
I have deleted it. Good catch, thanks :)